app/src/debug/java/org/simple/clinic/DebugClinicApp.kt

```diff
@@ -111,9 +111,9 @@ class DebugClinicApp : ClinicApp() {
           }
         })
         .bloodPressureModule(object : BloodPressureModule() {
-          override fun provideBloodPressureEntryConfig(): BloodPressureConfig {
-            return super.provideBloodPressureEntryConfig()
-                .copy(dateEntryEnabled = true)
+          override fun bloodPressureEntryConfig(): Single<BloodPressureConfig> {
```

Comment:
Should we make this an `Observable`?

Reply:
We should, but I did not want to put in effort in this PR :p

Reply:
update: since I'm going to make changes, I might as well do this

Reply:
Done